README.md

```diff
@@ -358,18 +358,21 @@ Hyperwallet.shared.retrieveTransferMethodConfigurationKeys(request: keysQuery) {
 
     guard let result = result else { return }
     // Get countries
-    let countryCodes = result.countries()
+    let countries = result.countries()
```

Comment:
You already have `countries` on line 365, don't you?

Reply:
Right.